src/coprocessor/metrics.rs

```diff
@@ -71,4 +71,18 @@ lazy_static! {
             &["type", "req"],
             vec![0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]
         ).unwrap();
+
+    pub static ref COPR_DIFF_EXEC_REQS: CounterVec =
+        register_counter_vec!(
+            "tikv_coprocessor_diff_exec_reqs",
```

Comment:
I think it would collect the number of  `executor` received, may be `CORP_EXECUTOR_COUNT_VEC` is ok ?